grabbit/tests/test_core.py

```diff
@@ -174,6 +185,11 @@ def test_get_nearest(self, layout):
         assert len(nearest) == 3
         assert nearest[0].subject == '01'
 
-    def test_exclude_regex(self, layout):
+    def test_index_regex(self, layout, layout2, layout_override):
```

Comment:
We should also ensure that if you pass both include and exclude, it raises an exception.